From 69e3bbf75975465e5503fbb0e6186c39884c645f Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Mon, 19 Aug 2024 21:20:22 +0800
Subject: [PATCH 2/2] Export python3 module
X-Developer-Signature: v=1; a=openpgp-sha256; l=1268; i=xtexchooser@duck.com;
 h=from:subject; bh=BJ4vcosZwdCqenJ39qhHHyk6FbXMCNwfuOqEHJ6Hhto=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGmH3ebz+lx6bDw7p9M+PqHM/XmpY6p69LYHvpq3ZAWXc
 fAE2JV0lLIwiHExyIopshQZNniz6qTziy4rl4WZw8oEMoSBi1MAJpIqyPDf06F0qujin2nLfk+L
 f2i6nVE/+PoxueWvNxsGcT1jjGyxZmRoZPm+wXfGH879u/oman8Rvjc3ZcP3FMkV8cbWvg0xJaL
 sAA==
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

Only a part of the PR is picked.

Link: https://github.com/AcademySoftwareFoundation/Imath/pull/400
---
 src/python/config/ModuleDefine.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/python/config/ModuleDefine.cmake b/src/python/config/ModuleDefine.cmake
index b51c6c10db65..7b90d5560c4a 100644
--- a/src/python/config/ModuleDefine.cmake
+++ b/src/python/config/ModuleDefine.cmake
@@ -126,7 +126,7 @@ function(PYIMATH_DEFINE_MODULE modname)
         LIBRARY_OUTPUT_NAME "${modname}"
         DEBUG_POSTFIX ""
       )
-      install(TARGETS ${modname}_python2 DESTINATION ${PyImath_Python2_SITEARCH_REL})
+      install(TARGETS ${modname}_python2 DESTINATION ${PyImath_Python2_SITEARCH_REL} EXPORT ${PROJECT_NAME})
     endif()
   else()
     if(TARGET Python3::Python AND TARGET Boost::${PYIMATH_BOOST_PY_COMPONENT})
@@ -153,7 +153,7 @@ function(PYIMATH_DEFINE_MODULE modname)
         LIBRARY_OUTPUT_NAME "${modname}"
         DEBUG_POSTFIX ""
       )
-      install(TARGETS ${modname}_python3 DESTINATION ${PyImath_Python3_SITEARCH_REL})
+      install(TARGETS ${modname}_python3 DESTINATION ${PyImath_Python3_SITEARCH_REL} EXPORT ${PROJECT_NAME})
     endif()
   endif()
 endfunction()
-- 
2.46.0

